app/views/recipes/show.html.haml

```diff
@@ -1,23 +1,34 @@
 %h3= @recipe.title
-%p= @recipe.description
-%p= @recipe.ingredients
-%p= @recipe.directions
 %p
   Posted by:
   = mail_to @recipe.user.email
+%p
+  - if @recipe.original_recipe_id.any?
```

Comment:
I would create an instance method on the `Recipe` class called `fork?` to determine if this is a fork we are dealing with, rather then having this logic in the view.

Reply:
:+1: